README.md

```diff
@@ -49,3 +51,15 @@ NOTICE:  version "X.X.X" of extension "postgis_tiger_geocoder" is already instal
 ALTER EXTENSION
 ```
 
+### Adding custom locale
+
+If you need to use another locale, just generate a new Dockerfile with this content (this example is for italian locale):
+
+```
+FROM mdillon/postgis:11
+RUN localedef -i it_IT -c -f UTF-8 -A /usr/share/locale/locale.alias it_IT.UTF-8
+ENV LANG it_IT.utf8
+```
+see more: [#136][i19]
+
+[i19]: https://github.com/appropriate/docker-postgis/issues/136
```

Comment:
This should reference the postgis github repo:
https://github.com/postgis/docker-postgis/issues/136